runescape-client/src/main/java/WorldMapData.java

```diff
@@ -443,6 +440,7 @@ static final int method343(int var0, int var1, int var2) {
       garbageValue = "263872365"
    )
    @Export("addChatMessage")
+   @Hook("addChatMessage")
```

Comment:
We can try this. Sometimes the obfuscator remove static methods and so the hook might not be mapped. But I think some methods are never removed.